packages/ui/src/components/switch/switch.tsx

```diff
@@ -0,0 +1,20 @@
+import * as RadixSwitch from '@radix-ui/react-switch';
+
+type Properties = {
+  value: boolean;
+  onChange: (value: boolean) => void;
+};
+
+export const Switch = ({ value, onChange }: Properties) => {
+  return (
+    <RadixSwitch.Root
+      className="group p-0.5 focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-indigo-500 focus-visible:ring-offset-2"
```

Comment:
what do you think about adding an alias to the group? If the parent element has its one it can impact this one.